en/components/lineargauge.md

```diff
@@ -0,0 +1,120 @@
+
+The main purpose of the Reserved area is to provide enough space for the numbering labels of the scale at any orientation – horizontal or vertical. The Reserved area automatically re-sizes when the orientation changes in order to accommodate the specific space requirements for displaying the numbering labels in each of the orientations.
+
+**Graph area** (2)  
+
+The area for displaying the needle, the tick marks, the ranges, and, optionally, the numbering labels of the linear gauge. All extent-related properties for these visual elements (except for labels) are measured against its edges. 
+
+Spread of the Graph area:
+
+**Along the scale** – Both the starting and ending positions of the scale are measured against the starting edge of the Graph area (the left edge at horizontal orientation or bottom edge at vertical orientation).
+
+**Across the scale** – the Graph area begins at the edge of the Reserved area (this is the bottom edge of the Graph area at horizontal orientation or its left edge at vertical orientation). The edge of the Graph area that borders with the Reserved area serves as reference point for the extent-related properties of some of the visual elements of the scale (for positioning these elements across the scale).
+
+## Configurable visual elements
+The linear gauge control features the following visual elements (See the picture below.)
+ 
+ ![](../images/Linear_Gauge_2.png)
+
+**Needle** (3) – This is the primary measure displayed by the control and is visualized as a bar.
+
+**Comparative range(s)** (4) – The ranges are visual elements that highlight a specified range of values on a scale. Their purpose is to visually communicate the qualitative state of the performance bar measure, illustrating at the same times the degree to which it resides within that state.
+
+**Tick marks** (5) –The tick marks serve as a visual division of the scale into intervals in order to increase the readability of the linear gauge.
+
+Major tick marks – The major tick marks are used as primary delimiters on the scale. The frequency they appear at, their extents and style can be controlled by setting the corresponding properties.
+
+Minor tick marks – The minor tick marks represent helper tick marks, which might be used to additionally improve the readability of the scale and can be customized in a way similar to the major ones.
+
+**Scale labels** (6) – The labels indicate the measures on the scale.
+
+**Border** (7) – The line visually delimiting the dimensions of the control.
+
+**Background** (8) – The background on which visual elements is placed is configurable in terms of pattern and color.
+ 
+**Tooltips** – show the values used to create the needle or the values, corresponding to the different ranges respectively.
```

Comment:
can you comment out tooltips for now? Tooltips are not working for any of gauges.